File: backends/go-gin/internal/domain/invitation/entity.go
package invitation

import "time"

// Invitation は招待のドメインエンティティです（GORM タグなし）。
type Invitation struct {
	ID        uint
	Token     string
	CreatedAt time.Time
	CreatedBy *uint
	UpdatedAt time.Time
	UpdatedBy *uint
	DeletedAt *time.Time
	DeletedBy *uint
	Version   int
}

File: backends/kotlin-ktor/src/main/kotlin/com/authorization/domain/invitation/Entity.kt
/*
 * 招待ドメインエンティティモジュール。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
package com.authorization.domain.invitation

import java.time.LocalDateTime

/**
 * 招待エンティティです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
data class Invitation(
    val id:        Int            = 0,
    val token:     String,
    val createdAt: LocalDateTime  = LocalDateTime.now(),
    val createdBy: Long?          = null,
    val updatedAt: LocalDateTime  = LocalDateTime.now(),
    val updatedBy: Long?          = null,
    val deletedAt: LocalDateTime? = null,
    val deletedBy: Long?          = null,
    val version:   Int            = 0,
)
